docs/organizations/managing-security-and-risk.md

```diff
@@ -104,6 +104,37 @@ The following table defines item priorities and days to fix the underlying Codac
 | Medium                  | Medium        | 90            |
 | Minor                   | Low           | 120           |
 
+## Access management<!-- Rename this. We're not really only managing access, are we? -->
+
+Organization owners can assign the Security Manager role to organization members, as well as revoke it. Security managers share some permissions with organization owners:
+
+|Permission|Organization Owner|Security Manager|
+|---|---|---|
+|Access Security and Risk Management|Yes|Yes|
+|Access originating items (Codacy repositories, Jira issues)|Yes<!-- TODO admin may nothave access here? Confirm -->|Inherits original permissions|
```

Comment:
Is it possible to clarify better what it means to access "originating items"? I'm thinking that it's navigating to the details of those items (or just viewing the item details), although I'm not sure how that will work with Jira items. :thinking:

Reply:
I just came across [this other line](https://github.com/codacy/docs/pull/1722/files#diff-30a735ab496ba1650575435a2195b32444487dbb269bb4b685b2e1ad8c7ebe53R37) and I should note that the two references should be kept in sync. :+1:

Reply:
This should link to either a filtered view of the Codacy issue list or directly to the Jira issue. I was looking for something generic enough to cover "Codacy issue", "Jira issue", and whatever we may include in the future.
```suggestion
|Access originating items<!-- TODO reword all "originating items" instances for clarity --> (Codacy repositories, Jira issues)|Yes<!-- TODO admin may not have access here? Confirm -->|Inherits original permissions|
```